lib/github.js

```diff
@@ -63,9 +63,10 @@ Organization.prototype.getRepositories = function(callback) {
 
     request.get(options, function (error, response, body) {
 
-        if (error) {
-            cli.error(error);
-            callback.call(this, error);
+        if (error || response.statusCode != 200) {
+            var message = (error || "") +
```

Comment:
Yes, we could simplify like you say, but in this case, if there is something in `error`, HTTP status data will not be displayed. But I think it should.